chucks-pick3-mobile/src/app/components/view-picks-card/view-picks-card.component.ts

```diff
@@ -41,6 +46,38 @@ export class ViewPicksCardComponent implements OnInit {
       }
     });
   }
-}
 
+  private retrieveNumbers() {
+    const generatedArray = this.drawTimeService.viewPicksArray;
+    const copiedArray = Object.assign([], generatedArray);
+    const ArrayToBeSetToTable: any = [];
+    for ( let i = 0; i < 12; i++){
+      ArrayToBeSetToTable.push(copiedArray.splice(Math.floor(Math.random() * copiedArray.length), 1));
+    }
+    this.savedArray = Object.assign([], copiedArray);
+    this.createTable(ArrayToBeSetToTable);
+  }
+  private createTable(ArrayToBeSetToTable) {
+    const newArray = Object.assign([], ArrayToBeSetToTable);
+    const myTableDiv = document.getElementById('myDynamicTable');
+    const table = document.createElement('TABLE');
+    const tableBody = document.createElement('TBODY');
+    table.appendChild(tableBody);
+
+    for (let i = 0; i < 4; i++) {
+      const tr = document.createElement('ion-row');
+      tableBody.appendChild(tr);
 
+      for (let j = 0; j < 3; j++) {
+        const generatedNumber = newArray.slice(0, 1);
+        const td = document.createElement('ion-col');
+        td.appendChild(document.createTextNode(generatedNumber));
+        tr.appendChild(td);
+        newArray.splice(0, 1);
+      }
+    }
+    myTableDiv.appendChild(table);
```

Comment:
This is a good first start; but we want to leverage angular ng-for and template design patterns.  Please re-factor.